### DISCUSSION TITLE
How to get a PR reviewed?

### DISCUSSION THREAD

**Discussion by aksestok** [dagster-io/dagster#19907] [Q&A] [answered]
A month back I uncovered some missing functionality in the `dagster-dbt` package and made a PR to include seeds when grabbing asset keys by model name in https://github.com/dagster-io/dagster/pull/19344

However, I haven't received a review or feedback on the PR, so I'm curious if there are any measures I can take in order to make that happen? If there are some guidelines or other best practices I have missed in order to get a review (other than the ones available on the ["Contributing" page](https://docs.dagster.io/community/contributing)), any feedback would be appreciated.

**Comment by garethbrickman** [comment, marked as answer]
I requested a reviewer for your PR 👍
